Makefile

```diff
@@ -2,6 +2,11 @@ BUILD_DIR=$(shell pwd)/build
 HANDLE_USER_DATA=$(shell base64 -w 0 scripts/handle-user-data)
 CERT_GEN=$(shell base64 -w 0 scripts/cert-gen.sh)
 VERSION=1.0.0-alpha.1
+GITTAG=$(shell git rev-parse --short HEAD)
+TODAY=$(shell date +"%d%m%Y%H%M%S")
+ifndef BUILD_ID
+    BUILD_ID=local
+endif
```

Comment:
Personally I would create the complete `build_id` string here and just use the variable in handle_user_data. Also I have one question, what would be the `BUILD_ID` for the GA build (e.g. CDK) in case we do not build it in a Jenkins job.